### DISCUSSION TITLE
Frontend Submit

### DISCUSSION THREAD

**Discussion by WXQ8NPb5TrbDNqzywcxF** [liedekef/events-made-easy#638] [General]
Hi,

I'm currently moving from old plugin to integrated Frontend Submit feature.
I've 2 problems.

I'm using templates.

1 - I'm unable to make a default value enabled for "event_seats"
#_FIELD{event_seats}{number}{value=2}
in html i have : <input type="number" id="event_seats" name="event[event_seats]" min="0" step="any" value="" value=2/><br>
I think correct syntax should be : `<input type="number" id="event_seats" name="event[event_seats]" min="0" step="any" value="2" /><br>`

2- I'm trying to set "location_id".  I've seen it's part of function eme_new_event.
#_FIELD{location_id}{hidden}{value=7}
html output : 
`<input type="hidden" id="location_id" name="event[location_id]"  /><input type="hidden" id="location_latitude" name="event[location_latitude]"  /><input type="hidden" id="location_longitude" name="event[location_longitude]"  /><br>`

I don't understand what i'm doing wrong. I think "default values" are not properly 

Thanks a lot for you help.

**Comment by liedekef** [comment]
1: never worked in the old form too I suspect :-) But this should fix it: https://github.com/liedekef/events-made-easy/commit/b4f77175a9a6e97ae18b53f365d3b9ee19f55352
2: I normally ignore location_id since I add it automatically when needed, but I understand the issue. This should fix it:
   https://github.com/liedekef/events-made-easy/commit/ace69ccdf21ecdfd5c8b254278594a4effdde274
